docs/06-troubleshooting.md

```diff
@@ -0,0 +1,47 @@
+# Troubleshooting
+
+## My DAC isn't working
+
+If you want to use your DAC with balena you have to follow this guide: [Click Me!](https://landr-balenalabs-repo-balena-sound-preview-landr.netlify.com/docs/dac-configuration/#configure-your-device-for-the-dac)
+
+If your DAC is not in the list can you ask us [in a issue](https://github.com/balenalabs/balena-sound/issues/new). We will then try to find the correct config for your DAC and add it to the list.
+
+## Missing RTP Packet error
+In some cases does the Pi give you errors saying there are RTP Packets missing.
```

Comment:
Fix:

"In some cases, the Pi gives you errors saying there are RTP Packets missing."